### DISCUSSION TITLE
command 'python.execInTerminal-icon' not found

### DISCUSSION THREAD

**Discussion by wabbyy** [coder/code-server#3003] [General] [answered]
System is Ubuntu 20.04.1

When put the button "run py file in terminal", it shows "command 'python.execInTerminal-icon' not found", how to fix it?

**Comment by wabbyy** [comment, marked as answer]
someone have same issue with me?

**Reply by kingdidierr** [reply]
Having the same issue on Ubuntu 20.04.1. Very confused. U figure anything out?

**Reply by wabbyy** [reply]
no i can not figure out,  could someone help us?

**Reply by kingdidierr** [reply]
i got a fix for now. I rolled back python extension on vs code to 2021.2.633441544. Just click the arrow next to uninstall and hit install another version.

**Reply by wabbyy** [reply]
@ kingdidierr
it works, thanks!

**Reply by kjs003** [reply]
yes I to have same problem

**Reply by rahulml25** [reply]
Thank you so much for this idea @wabbyy

**Reply by mahendrajakhar** [reply]
I was too lazy to follow the recommendations for reinstalling the program on the computer, so I went into the VSC extensions, uninstalled Python, restarted VSC, installed Python, restarted VSC and everything works now. If you are as lazy as I am, then you can try this method before taking drastic measures.
